pages/a/1k1-azure-post.html

```diff
@@ -0,0 +1,239 @@
+title: Crate guest post for Azure - Setting up Azure for Crate
+author: Chris Ward
+description: Setting up a 1001 node Crate cluster on Azure with Docker Swarm
+created: 2016-03-29
+status: publish
+post_type: post
+tags: azure, arm
+category: developernews
+
+# Crate guest post for Azure - Setting up Azure for Crate
+
+[Crate.io](https://crate.io/) is striving to create the most scalable database in our industry;
+we aim to make scaling your data from millions to billions of records and beyond, as simple as
+possible.
+
+We have clients who write and query gigabytes of data a minute, store terabytes a day and love the
+performance and stability that Crate offers. But the question always remained at the back of our
+minds: when taken to the extreme, what is Crate really capable of?
+
+We specifically wanted to:
+
+- Know how Crate would distribute shards and partitions across such a large cluster.
+- Examine the overhead added by the cluster management.
+- See at what speeds the cluster could write data.
+
```

Comment:
maybe add:

- Deploy a Crate cluster of >1k nodes
- Assess the performance of different configurations of the cluster